README.md

```diff
@@ -42,81 +42,165 @@ The policy models are neural networks that model the forward and backward transi
+
+```bash
+python main.py +experiments=grid/corners
+```
+
+Alternatively, we can explicitly indicate the environment and the proxy as follows:
 
-To better understand the GFlowNet components, let us explore the Scrabble environment in more detail below.
+```bash
+python main.py env=grid proxy=box/corners
+```
+
+The above command will train a GFlowNet with the default configuration, except for the environment, which will use `./config/env/grid.yaml`; and the proxy, which will use `./config/proxy/box/corners.yaml`.
+
+A typical use case of the gflownet library is to extend it with a new environment and a new proxy to fit your purposes. In that case, you could create their respective configuration files `./config/env/myenv.yaml` and `./config/proxy/myproxy.yaml` and run
+
+```bash
+python main.py env=myenv proxy=myproxy
+```
+
+The objective function to optimise is selected directly via the `gflownet` configuration. The following GFlowNet objectives are supported:
+
+- [Flow-matching (FM)](https://arxiv.org/abs/2106.04399): `gflownet=flowmatch`
+- [Trajectory balance (TB)](https://arxiv.org/abs/2201.13259): `gflownet=trajectorybalance`
+- [Detailed balance (DB)](https://arxiv.org/abs/2201.13259): `gflownet=detailedbalance`
+- [Forward-looking (FL)](https://arxiv.org/abs/2302.01687): `gflownet=forwardlooking`
 
-When initializing any GFlowNet agent, it's useful to explore the properties of the environment. The library offers various functionalities for this purpose. Below are some detailed examples, among others:
 
-1. Checking the Initial State 
+All other configurable options are handled similarly. For example, we recommend creating a user configuration file in `./config/user/myusername.yaml` specifying the directory for the log files in `logdir.root`. Then, it can be included in the command with `user=myusername` or `user=$USER` if the name of the YAML file matches our system username.
 
-You can observe the initial state of the environment. For Scrabble environment, this would be an empty board or sequence:
+As another example, you may also want to configure the functionality of the Logger, the class which helps manage logging to [Weights & Biases](https://wandb.ai/) during the training and evaluation of the model. Logging to WandB is disabled by default. In order to enable it, make sure to set up your WandB API key and set the configuration variable `logger.do.online` to `True` in your experiment config file or via the command line:
+
+```bash
+python main.py +experiments=grid/corners logger.do.online=True
+```
+
+Finally, also note that by default, PyTorch will operate on the CPU because we have not observed performance improvements by running on the GPU. You may run on GPU with `device=cuda`.
```

Comment:
You may have to be careful about this statement. I am not sure how performance is related to the CPU or GPU? if you have a big policy model, it will be unavoidable to use GPU :)

Reply:
I think the phrase "we have not observed..." is cautious enough, but  you're right in that we should investigate this in more detail.